packs/feats/alchemist-dedication.json

```diff
@@ -41,15 +41,40 @@
                 "path": "system.proficiencies.attacks.weapon-base-alchemical-bomb.rank",
                 "value": 1
             },
+            {
+                "itemUUID": "Compendium.pf2e.equipment-srd.Item.Versatile Vial",
+                "key": "SpecialResource",
+                "level": {
+                    "brackets": [
+                        {
+                            "end": 3,
+                            "value": 1
+                        },
+                        {
+                            "end": 11,
+                            "start": 4,
+                            "value": 4
+                        },
+                        {
+                            "end": 17,
+                            "start": 12,
+                            "value": 12
+                        },
+                        {
+                            "start": 18,
+                            "value": 18
+                        }
+                    ]
```

Comment:
no brackets

Reply:
a 4 step ternary if is kinda scary to do with brackets and I probably have some mild dyslexia as once the brackets nest that deep I can no longer read it out. How would you recommend going about it? AELikes?

I had a thought for a cascading if else function, something like `ifElse(cond1, result, cond2, result, cond3, result, result)` but I wasn't sure how amenable people would be to it.

Reply:
I ended up going with floorToValue(), its pushed up.